docs/source/tutorial.rst

```diff
@@ -303,15 +343,20 @@ We are working on giving the user more flexibility on this point.
             Rel. Error: 3.74044670e-04
     
 
-Note that the Monte Carlo methods are much more competitive for
-this case. The bad convergence properties of the trapezoid method are
-visible while Simpson’s rule is still OK given the comparatively smooth
-integrand.
+Note that the Monte Carlo methods are much more competitive in this case. 
+The bad convergence properties of the trapezoid method are visible while Simpson's 
+and Boole's rule are still OK given the comparatively smooth integrand. 
+
+Note that with the choice of sample points (3 per dimension), Boole's method cannot work. 
```

Comment:
I would change all to five then? Otherwise the comparison is pointless

Comparing VEGAS/MC with 59049 points to Boole with 9.765.625 is absurd

Reply:
But I cannot do this on my computer 😭

Reply:
Did it, see related ticket.